xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -132,6 +156,35 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     DocumentBuilder domBuilder = factory.newDocumentBuilder();
     domBuilder.setErrorHandler(new FatalErrorHandler());
     Element dom = domBuilder.parse(xml).getDocumentElement();
+
```

Comment:
What if someone is embedding xml that uses the same property replacement scheme, but draws properties from other locations... then we'd be getting failed replacements, and we have no way of knowing whats causing them - and whether they are valid or not (this would happen with Spring no?).  I think we should whitelist all "ehcache.org" namespaces for this replacement but leave other DOM elements alone.  We can make an unfiltered parameter subtitution method public so that if an extension wants to take advantage they can just call the method and pass their DOM fragment through it.